runelite-client/src/main/java/net/runelite/client/plugins/keyremapping/KeyRemappingListener.java

```diff
@@ -156,6 +156,11 @@ else if (config.esc().matches(e))
 				}
 			}
 
+			if (config.fkeyRemap() && plugin.isDialogOpen() && config.space().matches(e))
```

Comment:
Why is this tied to fkeys?

Reply:
Ah, yea I did that because the escape key was also tied into the fkeys, and because all the F-keys and escape key is underneath the "Remap F Keys" checkbox.

Reply:
But it is indeed a little bit confusing, maybe there should be an additional checkbox created for escape and space?

Reply:
The escape key is part of the fkeys as far as the game is concerned (you can f-key map a tab to escape)

Reply:
Ok so you'd prefer having the escape key under f-keys and just space underneath other?